Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "dapr"
+version = "0.1.0"
+authors = ["dapr.io"]
+edition = "2018"
```

Comment:
2019?

Reply:
The edition system was created for the release of [Rust 2018](https://doc.rust-lang.org/edition-guide/rust-2018/index.html).

Reply:
The edition system was created for the release of [Rust 2018](https://doc.rust-lang.org/edition-guide/rust-2018/index.html).

Reply:
The edition system was created for the release of [Rust 2018](https://doc.rust-lang.org/edition-guide/rust-2018/index.html).

Reply:
The edition system was created for the release of [Rust 2018](https://doc.rust-lang.org/edition-guide/rust-2018/index.html).